src/classes/AccountsSelector.cls

```diff
@@ -0,0 +1,94 @@
+public class AccountsSelector extends fflib_SObjectSelector {
+    
+    List<String> selectFields = new List<String>{'Name', 'BillingCity', 'BillingState', 'BillingCountry'};
```

Comment:
I'm not thinking why you always create an extra list of String in every selector class. It will not use in every selector class.